styles.css

```diff
@@ -0,0 +1,120 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
```

Comment:
[Optianal] It'll be better to use a flexbox for a container tag, you can remove it from the body so that you have a more independent body container for all sub-containers. What do you think?